packages/components/src/custom-gradient-picker/gradient-bar/control-points.tsx

```diff
@@ -73,9 +87,9 @@ function GradientColorPickerDropdown( {
 	isRenderedInSidebar,
 	className,
 	...props
-} ) {
+}: WordPressComponentProps< CustomColorPickerDropdownProps, 'div', true > ) {
 	// Open the popover below the gradient control/insertion point
-	const popoverProps = useMemo(
+	const popoverProps: DropdownProps[ 'popoverProps' ] = useMemo(
```

Comment:
We could use `as const` to narrow down the type, and therefore avoid the need of using `DropdownProps`

```tsx
const popoverProps = useMemo(
		() =>
			( {
				placement: 'bottom',
				offset: 8,
			} as const ),
		[]
	);
```

Reply:
👍 0f56c0b5a9